content/c-sharp/concepts/system-math/system-math.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'System.Math'
+Description: 'Common mathematical functions can be performed using the System.Math class.'
+Subjects: 
+- 'Code Foundations'
+- 'Computer Science'
+Tags: 
+- 'Libraries'
+- 'Arithmetic'
+- 'Functions'
+CatalogContent: 
+- 'learn-c-sharp'
+- 'paths/build-web-apps-with-asp-net'
+---
+
+When writing code in C#, common mathematical functions can be performed using the **System.Math** class. System.Math is a static class that is included within .NET as part of the `System` namespace. [.NET](https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet) is a free, cross-platform, open source developer platform created by Microsoft.
+
+## Syntax
+
+The using directive `using System;` can be included within a .cs file to gain access to the Math class.
+
+The Math class is static and is accessed without needing to create an instance.
+
+## Example
+
+```cs
+//include the System namespace within xxx.cs
+using System;
+
+//elsewhere within xxx.cs
+int x = Math.Pow(4,2); //raise 4 to the 2nd power
```

Comment:
Can we add a line printing out the variable to make it more interactive? Also Math.Pow returns a "double".
```suggestion
public class Example
{
  public static void Main(string[] args)
  {
      // Raise 4 to power of 2
      double x = Math.Pow(4,2);
      
      Console.WriteLine("x = {0}", x);
      // Output: x = 16
  }
}
```